### PR TITLE
fix analysis and disease id references

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by WendelHime — CHANGES_REQUESTED** [review]
Review some things

File: Analysis.py [hunk 1]
```diff
@@ -18,9 +18,7 @@ class Analysis(Base.Base):
     def __init__(self,
                  id=0,
                  idImage=0,
-		 image=object(),
-		 idClassifier=0,
-		 classifier=object(),
```
[the remark this answers is not in the document]
Comment (BiondiVini) [reply Analysis.py]: Sorry, I will revert this commit

File: Analysis.py [hunk 2]
```diff
@@ -2,31 +2,34 @@
 from sqlalchemy.orm import relationship
 import database.Image
 import database.Classifier
+import database.AnalysisResult
 from models.Analysis import Analysis as AnalysisModel
 from repository.base import Base
 
 class Analysis(Base.Base):
-    __tablename__ = 'analysis'
+    __tablename__ = 'analyzes'
```
Comment (WendelHime) [inline Analysis.py]: It should be analyses or you can keep the way as it was

**Review by WendelHime — CHANGES_REQUESTED** [review]
Update table reference

**Review by WendelHime — APPROVED** [review]
Everything appears ok

**Pull request merged**